src/Home.js

```diff
@@ -59,6 +60,9 @@ class Home extends Component<Props, State> {
     let dataContainer = (
       <div className="bottomLeftContainer">
         <DataContainer />
+        <div className="darkMode">
+          <DarkMode />
+        </div>
```

Comment:
There shouldn't be a need for this extra `<div>` if you move the CSS into the `DarkMode` component.